ExampleApp/src/main/java/co/infinum/povexampleapp/CommonUsageExample.java

```diff
@@ -87,7 +87,7 @@ protected void onCreate(Bundle savedInstanceState) {
         initUI();
 
         /*  create new instance of updater associated with application context   */
-        updater = new PrinceOfVersions(this);
+        updater = new PrinceOfVersions(getApplicationContext());
```

Comment:
This really is a major issue.
Provided context is set to a static field in ContextHelper and the Activity is leaked throughout the whole app lifecycle.